.github/workflows/build_and_test.yml

```diff
@@ -227,8 +227,6 @@ jobs:
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DSLEEF_ENFORCE_RVVM1=ON -DSLEEF_ENFORCE_RVVM2=ON"
             # Disable inline headers, they just don't compile on riscv64
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DSLEEF_BUILD_INLINE_HEADERS=OFF"
-            # Disable dft, it fails with linker error to `cexp`
-            EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DSLEEF_BUILD_DFT=OFF"
             # Disable quad, it's missing the `Sleef_quad` function
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DSLEEF_BUILD_QUAD=OFF"
```

Comment:
If you're adding support for quad at https://github.com/shibatch/sleef/pull/503/files#diff-cced2a666d218fe891973b1e7d9ef8b64c4467b511ab815073c4fc670bc91a31R1152, you will also need to remove this line to make sure it's tested on CI.

Reply:
I moved the intrinsics implementations for quad into this patch by accident, but then decided I might as well leave them there (at the risk of making this an ever-growing PR) so that we don't have two rounds of reviewing the intrinsics implementation themselves.

I was going to say, I have a bigger [patch for quad](https://github.com/rivosinc/sleef/compare/dev/shosie/fix-inline-headers...rivosinc:sleef:dev/shosie/fix-rvv-quad) but looking at it now, I might just as well cherry-pick that into this patch, because it doesn't look like there's much left to object to.

The only problem is that the order I did things, those patches were written _after_ [inline-header support](https://github.com/rivosinc/sleef/compare/dev/shosie/fix-rvv-dft...rivosinc:sleef:dev/shosie/fix-inline-headers).  My approach there is more questionable.  Mostly because of the complexity of handling the incompatibility between SP and DP implementations of some functions.

The size of that PR has also become much smaller after rework, here, but it still has the same design issues (eg., emitting conditional preprocessor logic in the generated macroonly*.h files).

Let's see if I can't reorder the patches and fold the rest of quad into this....

Reply:
Looks like I can indeed!